lld/ELF/Arch/BPF.cpp

```diff
@@ -30,7 +30,9 @@ class BPF final : public TargetInfo {
                      const uint8_t *loc) const override;
   RelType getDynRel(RelType type) const override;
   int64_t getImplicitAddend(const uint8_t *buf, RelType type) const override;
-  void relocate(uint8_t *loc, const Relocation &rel, uint64_t val) const override;
```

Comment:
If there is a bunch of pure formatting changes it's nice to pull those into a separate pr

Reply:
Removed